crates/bevy_sprite/src/render/mod.rs

```diff
@@ -178,6 +181,45 @@ impl SpecializedPipeline for SpritePipeline {
     }
 }
 
+pub fn calculate_bounds(
```

Comment:
This needs doc strings: it's a public system.

Reply:
Ups...that's an oversight, the system doesn't need to be public.

Reply:
Hm...well maybe not exactly an oversight, because all the other functions in that file are `pub` too. But i can make it `pub(crate) calculate_bounds` if you want.